runners_manager/runner/tests/tests_runner.py

```diff
@@ -95,9 +94,15 @@ def test_config_vm_type(self, name_generator: Mock,
         self.assertEqual(r.runner_management.__len__(), 1)
 
         self.assertEqual(r.runners.__len__(), 2)
-        self.assertEqual(r.runners['0'], Runner('0', '1', r.runner_management[0]))
-        self.assertEqual(r.runners['1'].__dict__,
-                         Runner('1', '2', r.runner_management[0]).__dict__)
+        # Runner('0', '1', r.runner_management[0])
```

Comment:
Either remove or uncomment this line. Otherwise we will forget what to do with it :D